pkg/scheduler/cache/cache.go

```diff
@@ -462,8 +462,8 @@ func (cache *schedulerCache) ListPDBs(selector labels.Selector) ([]*policy.PodDi
 }
 
 func (cache *schedulerCache) IsUpToDate(n *NodeInfo) bool {
-	cache.mu.Lock()
-	defer cache.mu.Unlock()
+	cache.mu.RLock()
```

Comment:
The lock in the above function (ListPDBs) should also be RLock.

Reply:
Oh, nice catch, just fixed